pkg/operator/staticresourcecontroller/static_resource_controller.go

```diff
@@ -71,6 +71,71 @@ func NewStaticResourceController(
 	return c
 }
 
+func (c *StaticResourceController) AddKubeInformers(kubeInformersByNamespace v1helpers.KubeInformersForNamespaces) *StaticResourceController {
```

Comment:
@deads2k this is also ruining feng-shui of my controller factory...

Reply:
> @deads2k this is also ruining feng-shui of my controller factory...

heh, I'm afraid it is pretty essential.  Does it just need a rename?

Reply:
no, I will have to rework this after this merge in factory PR... basically you register just informers to factory and then factory add event handlers and take care of cache... don't worry about it now.